packages/phoenix-ng/cypress/e2e/index.cy.ts

```diff
@@ -0,0 +1,130 @@
+    cy.get('footer > p > a')
+      .eq(0)
+      .should('have.attr', 'href', 'https://github.com/HSF/phoenix');
+    cy.get('footer > p > a')
+      .eq(1)
+      .should(
+        'have.attr',
+        'href',
+        'https://github.com/HSF/phoenix/blob/master/README.md'
+      );
+  });
+
+  describe('Test the card body', () => {
+    it('should display the Playground card properly', () => {
+      cy.get('.card-body').eq(0).should('be.visible');
+      cy.get('.card-title').eq(0).contains('Playground');
+      cy.get('.card-text')
+        .eq(0)
+        .contains('Get started with the different Phoenix features')
+        .should('exist');
+      cy.get('.btn-primary').eq(0).contains('Show').should('exist');
+    });
+
+    it('should display the Geometry display card properly', () => {
+      cy.get('.card-body').eq(1).should('be.visible');
+      cy.get('.card-title').eq(1).contains('Geometry display').should('exist');
+      cy.get('.card-text')
+        .eq(1)
+        .contains('This test should show some simple geometry')
+        .should('exist');
+      cy.get('.btn-primary').eq(1).contains('Show').should('exist');
+    });
+
+    it('should display the ATLAS card properly', () => {
+      cy.get('.card-body').eq(2).should('be.visible');
+      cy.get('.card-title').eq(2).contains('ATLAS').should('exist');
+      cy.get('.card-text')
+        .eq(2)
+        .contains('Show the ATLAS detector. One simple event')
+        .should('exist');
+      cy.get('.btn-primary').eq(2).contains('Show').should('exist');
+    });
+
+    it('should display the LHCb card properly', () => {
+      cy.get('.card-body').eq(3).should('be.visible');
+      cy.get('.card-title').eq(3).contains('LHCb').should('exist');
+      cy.get('.card-text')
+        .eq(3)
+        .contains('Show the LHCb detector. One simple event')
+        .should('exist');
+      cy.get('.btn-primary').eq(3).contains('Show').should('exist');
+    });
+
+    it('should display the CMS card properly', () => {
+      cy.get('.card-body').eq(4).should('be.visible');
+      cy.get('.card-title').eq(4).contains('CMS').should('exist');
+      cy.get('.card-text')
+        .eq(4)
+        .contains('Show the CMS detector. One simple event')
+        .should('exist');
+      cy.get('.btn-primary').eq(4).contains('Show').should('exist');
+    });
+
+    it('should display the TrackML card properly', () => {
+      cy.get('.card-body').eq(5).should('be.visible');
+      cy.get('.card-title').eq(5).contains('TrackML').should('exist');
+      cy.get('.card-text')
+        .eq(5)
+        .contains(
+          'Visualisation for TrackML. Shows how to write a custom event loader'
+        )
+        .should('exist');
+      cy.get('.btn-primary').eq(5).contains('Show').should('exist');
+    });
+
+    it('should navigate to the correct page', () => {
+      cy.get('.btn-primary').eq(0).click();
+      cy.url().should('eq', 'http://localhost:4200/#/playground');
```

Comment:
> unnecessarily checking if elements exist

Right. Instead of checking for elements in DOM, I should make sure to check whether they are visible or not.

> what e2e tests actually are.

I was not sure earlier up to what extent I should test the workflow. But, I'm adding more scenarios now. 👍